docs/concepts/login.md

```diff
@@ -1,2 +1,74 @@
 (login)=
-# {octicon}`sign-in` Login
\ No newline at end of file
+# {octicon}`sign-in` Login
+
+Finetuner, as part of the [Jina AI](https://jina.ai/) stack,
+uses [Jina AI Cloud](https://cloud.jina.ai/) for authentication.
+
+Logging in via Jina AI Cloud is very straightforward.
+From Python, all it takes is the following two lines of code:
+
+```python
+import finetuner
+
+finetuner.login()
+```
+
+Once you call the `~finetuner.login()` method,
+a new tab will open in your browser, offering several login options.
+After completing the login process, you will receive the following message in your terminal:
+
+```bash
+Your browser is going to open the login page.
+If this fails please open the following link: ...
+🔐 Successfully logged in to Jina AI as [YOUR-USERNAME] (username: your-username)!
+```
+
+```{admonition} What happens when I login?
+:class: hint
+Finetuner will generate a temporary `auth_token` that is associated with your personal account.
+This `auth_token` will be utilized by Finetuner to carry out various tasks such as:
+
+1. Sending your training data to [Jina AI Cloud Storage](https://cloud.jina.ai/user/storage).
+2. Setting up GPU/CPU resources for performing fine-tuning tasks.
+3. Storing fine-tuned models (referred to as Artifacts) on the Jina AI Cloud.
+4. Performing billing (applicable only when the user exhausts their free resources).
+
+By default, your data and fine-tuned models are visible only to you unless you publish them by manually setting the `public` flag to `True`.
+```
+
+## Namespace
+
+Jina AI Cloud users automatically have a namespace linked to their accounts.
+This ensures that even if two users give some artifact or resource the same name,
+they can be separately addressed.
+
+If you want to download public data or models published by other users,
+you must add `[namespace]/` before the name of the data or artifact.
+For example, `finetuner/quora-train-da`,
+where `finetuner` is the namespace for the account, and `quora-train-da` is the name of the dataset,
+which refers to the quora training set.
```

Comment:
```suggestion
Public resources are in the `finetuner` namespace. For example, to use the Quora training dataset, use the name  `finetuner/quora-train-da`.
```

Also, link "Quora training dataset" to some reference for that dataset. I don't know which one it is.

Reply:
done